llama_stack/providers/utils/memory/vector_store.py

```diff
@@ -174,56 +184,35 @@ async def delete(self):
 
 
 @dataclass
-class BankWithIndex:
-    bank: VectorMemoryBank
+class VectorDBWithIndex:
+    vector_db: VectorDB
     index: EmbeddingIndex
     inference_api: Api.inference
 
-    async def insert_documents(
+    async def insert_chunks(
         self,
-        documents: List[MemoryBankDocument],
+        chunks: List[Chunk],
     ) -> None:
-        for doc in documents:
-            content = await content_from_doc(doc)
-            chunks = make_overlapped_chunks(
```

Comment:
this is now removed from here and will be moved to the "memory tool" in the next PR